src/screens/dashboard/DashboardScreen.tsx

```diff
@@ -33,7 +41,7 @@ export class DashboardScreen extends React.PureComponent<NavigationInjectedProps
         <FullScreenTemplate padded darkBackground>
           {student && <StudentPlanList student={student} />}
         </FullScreenTemplate>
-        <CreateStudentButton />
+        {student && <CreateButton actionName="create student" onPress={Student.create} />}
```

Comment:
As mentioned in previous code review please do not use whitespaces in such strings. Use `create-student` or any other character than whitespace.